src/python_inspector/resolution.py

```diff
@@ -771,6 +772,9 @@ def get_resolved_dependencies(
         )
     except Exception as e:
         if verbose:
-            import click
+            if ctx:
+                import click
 
-            click.secho(f"{e!r}", err=True)
+                click.secho(f"{e!r}", err=True)
```

Comment:
For this rather than to have an import inside the code, I would rather pass as an argument a "printer" function/callable that is either a curried version of `secho = functools.partial(click.secho, err=True)` or a plain `print` this way the function never deal with ctx or else. Instead the default is to use `print` and you can  call with your click alternate.

So the function args would gain: `printer=print` as a new arg.